src/c/datove_typy/celociselne_typy.md

```diff
@@ -22,16 +22,16 @@ typ, pokud použijeme `unsigned`, tak použijeme typ bez znaménka. Většina ty
 znaménkem, tj. `int` je to samé jako `signed int`. V následující tabulce je seznam nejčastějších
 celočíselných typů[^2]:
 
-| Název | Počet bytů | Rozsah hodnot | Se znaménkem |
-|---|:---:|:---:|:---:|
-| `char` nebo<br />`signed char` | 1 | \[-128; 127\] | <i class="fa fa-check"></i> |
-| `unsigned char` | 1 | \[0; 255\] | <i class="fa fa-times"></i> |
-| `short` nebo<br />`signed short` | 2 | \[-32 768; 32 767\] | <i class="fa fa-check"></i> |
-| `unsigned short` | 2 | \[0; 65 535\] | <i class="fa fa-times"></i> |
-| **`int`** nebo<br />`signed int` | 4 | \[-2 147 483 648; 2 147 483 647\] | <i class="fa fa-check"></i> |
-| `unsigned int` | 4 | \[0; 4 294 967 295\] | <i class="fa fa-times"></i> |
-| `long` nebo<br />`signed long` | 8 | \[-9 223 372 036 854 775 808;<br />9 223 372 036 854 775 807\] | <i class="fa fa-check"></i> |
-| `unsigned long` | 8 | \[0; 18 446 744 073 709 551 615] | <i class="fa fa-times"></i> |
+| Název                            | Počet bytů |                         Rozsah hodnot                          |        Se znaménkem         |
+|----------------------------------|:----------:|:--------------------------------------------------------------:|:---------------------------:|
+| `char` nebo<br />`signed char`   |     1      |                         \[-128; 127\]                          | <i class="fa fa-check"></i> |
+| `unsigned char`                  |     1      |                           \[0; 255\]                           | <i class="fa fa-times"></i> |
+| `short` nebo<br />`signed short` |     2      |                      \[-32 768; 32 767\]                       | <i class="fa fa-check"></i> |
+| `unsigned short`                 |     2      |                         \[0; 65 535\]                          | <i class="fa fa-times"></i> |
+| **`int`** nebo<br />`signed int` |     4      |               \[-2 147 483 648; 2 147 483 647\]                | <i class="fa fa-check"></i> |
```

Comment:
Proc je toto v boldu?

Reply:
(Není to nová změna, bylo to tak už předtím). Dal jsem to do boldu asi proto, abych dal najevo, že je to takový "defaultní" typ. Je to okomentováno i v textu pod tabulkou.

Reply:
OK, my bad.